consensus/service/src/api/client_api_service.rs

```diff
@@ -199,6 +224,38 @@ impl ConsensusClientApi for ClientApiService {
             Ok(response)
         });
 
+        mc_common::logger::scoped_global_logger(&rpc_logger(&ctx, &self.logger), |logger| {
+            send_result(ctx, sink, result, logger)
+        });
+    }
+    fn propose_mint_tx(
```

Comment:
Nit: Add blank line between methods
```suggestion

    fn propose_mint_tx(
```